python/ray/serve/batching.py

```diff
@@ -242,6 +248,33 @@ def __del__(self):
         self._handle_batch_task.cancel()
 
 
+def _validate_max_batch_size(max_batch_size):
+    if not isinstance(max_batch_size, int):
+        if isinstance(max_batch_size, float) and max_batch_size.is_integer():
+            max_batch_size = int(max_batch_size)
+        else:
+            raise TypeError(
+                "max_batch_size must be integer >= 1"
```

Comment:
Can you include what the `max_batch_size` is in this error? If it gets updated to an invalid value by the magic functions, it would be useful to know what that value is.

Reply:
Same feedback for the other assertions in this function and `_validate_batch_wait_timeout_s`.